src/wiring_diagrams/ScheduleUndirected.jl

```diff
@@ -0,0 +1,219 @@
+""" Scheduling and evaluation of undirected wiring diagrams.
+
+In category-theoretic terms, this module is about evaluating arbitrary
+composites of morphisms in hypergraph categories.
+"""
+module ScheduleUndirectedWiringDiagrams
```

Comment:
This is an insanely long module name. Could we call it Schedules since it already lives in UndirectedWiringDiagrams and the import path would be `Catlab.WiringDiagrams.Schedules`?